### PR TITLE
Optionally upgrade DB if installing a new server but using an existing DB, add new web requirements

### PR DESCRIPTION
**Pull request by manics**
`omego install [--upgradedb|--initdb] ...`

Also pip install django and gunicorn

### CODE REVIEW

File: omero-deploy/Dockerfile [hunk 1]
```diff
@@ -12,7 +12,10 @@ RUN apt-get update -y && \
     sudo \
     vim
 
-RUN /opt/anaconda/bin/pip install omego
+RUN /opt/anaconda/bin/pip install \
+    omego \
+    "django<1.9" \
```
Comment (joshmoore) [inline omero-deploy/Dockerfile]: Thoughts on using the requirements file? This seems brittle. cc: @sbesson

Comment (manics) [inline omero-deploy/Dockerfile]: Could do, but since this is a test-image it'll have to be done at runtime after installing the server.

Comment (joshmoore) [inline omero-deploy/Dockerfile]: Guess longer-term, `omego` could be responsible for that, but we're not even close. Perhaps just an RFE then?

Comment (manics) [inline omero-deploy/Dockerfile]: Yes, my priority at the moment is to get stuff working

**Pull request merged**